tools/openblas_support.py

```diff
@@ -0,0 +1,310 @@
+import os
+import sys
+import glob
+import shutil
+import textwrap
+import platform
+import hashlib
+
+from tempfile import mkstemp, gettempdir
+from urllib.request import urlopen, Request
+from urllib.error import HTTPError
+import zipfile
+import tarfile
+
+OPENBLAS_V = 'v0.3.9'
+OPENBLAS_LONG = 'v0.3.9'
```

Comment:
This is a pretty old openblas version; there's 0.3.15 in the meantime, see [here](https://github.com/xianyi/OpenBLAS/releases)